cms/templates/js/content-visibility-editor.underscore

```diff
@@ -0,0 +1,33 @@
+<form>
+<h3 class="modal-section-title" id="content_visibility_label"><%- gettext('Student Visibility') %></h3>
+<div class="modal-section-content staff-lock">
+    <ul class="list-fields list-input content-visibility" role="group" aria-labelledby="content_visibility_label">
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="content-visibility" type="radio" value="staff_only">
+                <%- gettext('Hide from students') %>
+            </label>
+        </li>
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="content-visibility" type="radio" value="hide_after_due">
+                <%- gettext('Hide content but not grades from students after due date') %>
+            </label>
+        </li>
+        <li class="field-radio">
+            <label class="label">
+                <input class="input input-radio" name="content-visibility" type="radio" value="visible">
+                <%- gettext('Always show to students (Default)') %>
+            </label>
+        </li>
+    </ul>
+
+    <% if (hasExplicitStaffLock && !ancestorLocked) { %>
+        <p class="tip tip-warning">
+            <% var message = gettext('If you make this %(xblockType)s visible to students, students will be able to see its content after the release date has passed and you have published the unit.'); %>
```

Comment:
Thanks @efischer19. My brain found it difficult to change viewpoints from the "Hide" action offered by our options to the opposite "make visible" action referred to in the tip. Also, the "clear option" reference is confusing since we are not using checkboxes.
If I understand you correctly, we follow the pattern but don't use the same message text? If so, let's work out some alternative message text. I still need to think through the context-sensitive scenarios...